File: jupyter2/ps/function/projectile_height/jl/projectile_height.jl
### begin my answer

### begin hidden
height(h0, v0, t) = h0 + v0 * t - 0.5 * 9.8 * t * t
### end hidden
### end my answer

function main()
    @assert abs(height(10,  1, 1) - 6.1) < 1e-5
    @assert abs(height(20,  0, 2) - 0.4) < 1e-5
    @assert abs(height(30, -1, 3) - -17.1) < 1e-5
    println("OK")
end

main()

File: jupyter2/ps/function/projectile_height/rs/projectile_height.rs
/** begin my answer */

/** begin hidden */
fn height(h0: f64, v0: f64, t: f64) -> f64 {
    h0 + v0*t - 0.5*9.8*t*t
}
/** end hidden */
/** end my answer */

fn main() {
    assert!((height(10.0,  1.0, 1.0) -   6.1).abs() < 1e-5);
    assert!((height(20.0,  0.0, 2.0) -   0.4).abs() < 1e-5);
    assert!((height(30.0, -1.0, 3.0) - -17.1).abs() < 1e-5);
    println!("OK");
}
